### ISSUE TITLE
Make <div id="main-container"> the main landmark

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#838]
Part of #820

I think this is a Blacklight-level change. It's WCAG best practice that pages have a main landmark, and that all content on the page is contained by landmarks. 

- [x] Change 
    `<div id="main-container" class="container">` to 
    `<main id="main-container" class="container" role="main" aria-label="Main content">`
